Source/OxyPlot/PlotModel/PlotModel.cs

```diff
@@ -193,6 +193,16 @@ public enum TitleHorizontalAlignment
     public partial class PlotModel : Model, IPlotModel
     {
         /// <summary>
+        /// The snap timer.
+        /// </summary>
+        private readonly Timer snapTimer;
```

Comment:
I don't like adding a timer. I think we discussed this before, must test. How will this work if you are printing the `PlotModel`? Should you wait a few timer intervals before printing? :-)